### PR TITLE
initial update for v 5.2.3

### PR DESCRIPTION
**Pull request by AdMax18**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by AdMax18** [comment]
@conda-forge-admin, please rerender

**Comment by AdMax18** [comment]
bumping for review and merge @igortg @kwilcox  @mcs07

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,61 +19,30 @@ build:
...
+    - kombu >=5.2.3,<6.0
+    - python >=3.7
+    - pytz >=2021.3
+    - setuptools >=59.1.1,<59.7.0
     - vine >=5.0.0,<6.0
-  run_constrained:
-    - sphinx >=2.0
-
+  
 test:
```
Comment (chrisburr) [inline recipe/meta.yaml:37]: Can you add the tests back here?

**Review by chrisburr — CHANGES_REQUESTED** [review]

**Comment by AdMax18** [comment]
@conda-forge-admin, please rerender

**Comment by AdMax18** [comment]
@conda-forge-admin, please rerender

**Comment by chrisburr** [comment]
Thanks @AdMax18! 🍰

**Pull request merged**